app/models/item.rb

```diff
@@ -1,3 +1,9 @@
 class Item < ApplicationRecord
   belongs_to :supermarket
-end
\ No newline at end of file
+  has_many :customer_items
+  has_many :customers, through: :customer_items
+
+  def customer_count
+    customers.count
```

Comment:
Great work encapsulating this calculation in a model method! I know it's a simple calculation, but according to strict MVC convention, it's still a data query beyond a simple CRUD method so it warrants its own model method!